src/control-plane/backend/click-stream-api.ts

```diff
@@ -503,7 +502,7 @@ export class ClickStreamApiConstruct extends Construct {
             new iam.PolicyStatement({
               effect: iam.Effect.ALLOW,
               resources: [
-                `arn:${Aws.PARTITION}:quicksight:*:${Aws.ACCOUNT_ID}:dashboard/Clickstream*`,
+                `arn:${Aws.PARTITION}:quicksight:*:${Aws.ACCOUNT_ID}:dashboard/clickstream*`,
```

Comment:
Is it required for the action `GenerateEmbedUrlForRegisteredUser`? We should meet the permission issue if granting wrong dashboards?

If it's required, should we extract `clickstream` as a constant or use an existing constant?

Reply:
use `QUICKSIGHT_RESOURCE_NAME_PREFIX`